monitor/README.md

```diff
@@ -1,5 +1,6 @@
 # Bisq Network Monitor Node
 
+
```

Comment:
Could you please remove this empty line? Thanks!

Reply:
Done.